api/rest.go

```diff
@@ -2008,6 +2008,10 @@ func handleContentUnitsList(cp utils.ContextProvider, exec boil.Executor, r Cont
 	}
 	appendPublishedFilterMods(&mods, r.PublishedFilter)
 
+	if r.OriginalLanguage != "" {
```

Comment:
please keep the `append<Filter name goes here>Mods(...)` style. 
So this should be in a method for this new filter

Reply:
was not sure tha its common situation (properties->>'original_language')
but was add append

Reply:
added